Automattic/wp-calypso
# Use bulk fixer endpoints and improve feedback with notices

#### Changes proposed in this Pull Request

* Use bulk fixer endpoints instead of the individual ones.
* Improve UI feedback for fixers by pooling the bulk fixer status endpoint and showing a notice in case of success or error.

#### Testing instructions



* Set up a JP site with Scan and add a few threats (You can use [this plugin](https://github.com/Automattic/jetpack-threat-tester) to add a few innocuous threats for testing).
* Go to the scan section of both Calypso Blue and Calypso Green (To test Calypso Green you need to run `yarn start start-jetpack-cloud` and then go to `http://jetpack.cloud.localhost:3000` on your browser)  and try out the fix all button and verify that a single network call was made.
* Verify that a success/error notice appears when all fixers finish their processing.

#### Screenshots 
Before we showed no feedback in case the fixer didn't work:
![error-old](https://user-images.githubusercontent.com/12788275/97618805-42eabe00-19fe-11eb-9386-9eff5c046271.gif)
After we start showing a notice in case one of the fixers failed:
![error](https://user-images.githubusercontent.com/12788275/97609825-dae2aa80-19f2-11eb-87a0-02cc5b01c010.gif)
